packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -493,7 +493,7 @@ async function handler(
 
   let routingFormResponse = null;
 
-  if (routedTeamMemberIds) {
+  if (routedTeamMemberIds && routingFormResponseId) {
```

Comment:
@emrysal I'm not sure if this the expected behavior tho. But Sentry is erroring because of undefined `routingFormResponseId`